zoo-css-challenge/style.css

```
@@ -166,18 +202,29 @@ footer {
   background-color: rgba(255, 255, 255, 0.75);
   padding: 50px;
   max-width: 400px;
+  border-radius: 8px;
 }
 
 .education-container h3,
 .education-container p {
   margin-bottom: 1em;
 }
 
+ .education-container button {
+  font-size: 120%;
+ }
+ 
+ .education-container button:focus, button:hover {
```

```suggestion
 .education-container button:focus, .education-container button:hover {
```
The second selector here `button:hover` will apply to all hovered buttons. Instead, we need to change it to `.education-container button:hover` so that the styles only apply to hovered buttons in the education section

---

That makes sense, thanks!